themes/src/main/resources-community/theme/base/login/messages/messages_pt_BR.properties

```diff
@@ -202,13 +264,108 @@ invalidCodeMessage=C\u00F3digo inv\u00E1lido, por favor fa\u00E7a login novament
+identityProviderAlreadyLinkedMessage=A identidade federada retornada por {0} j\u00E1 est\u00E1 vinculada a outro usu\u00E1rio.
+confirmAccountLinking=Confirme a vincula\u00E7\u00E3o da conta {0} do provedor de identidade {1} \u00E0 sua conta.
+confirmEmailAddressVerification=Confirme a validade do endere\u00E7o de e-mail {0}.
+confirmExecutionOfActions=Execute a(s) seguinte(s) a\u00E7\u00E3o(\u00F5es)
+
+locale_ca=Catal\u00E0
+locale_de=Deutsch
+locale_en=English
+locale_es=Espa\u00F1ol
+locale_fr=Fran\u00E7ais
+locale_it=Italiano
+locale_ja=\u65E5\u672C\u8A9E
+locale_nl=Nederlands
+locale_no=Norsk
+locale_pl=Polish
+locale_pt_BR=Portugu\u00EAs (Brasil)
+locale_pt-BR=Portugu\u00EAs (Brasil)
+locale_ru=\u0420\u0443\u0441\u0441\u043A\u0438\u0439
+locale_lt=Lietuvi\u0173
+locale_zh-CN=\u4E2D\u6587\u7B80\u4F53
+locale_sk=Sloven\u010Dina
+locale_sv=Svenska
 
 backToApplication=&laquo; Voltar para o aplicativo
-missingParameterMessage=Par\u00E2metros que faltam\: {0}
+missingParameterMessage=Par\u00E2metros ausentes\: {0}
 clientNotFoundMessage=Cliente n\u00E3o encontrado.
+clientDisabledMessage=Cliente desativado.
 invalidParameterMessage=Par\u00E2mentro inv\u00E1lido\: {0}
-alreadyLoggedIn=You are already logged in.
+alreadyLoggedIn=Voc\u00EA j\u00E1 est\u00E1 logado.
+differentUserAuthenticated=Voc\u00EA j\u00E1 est\u00E1 autenticado como usu\u00E1rio diferente ''{0}'' nesta sess\u00E3o. Por favor, saia primeiro.
+brokerLinkingSessionExpired=Vincula\u00E7\u00E3o de conta do broker solicitada, mas a sess\u00E3o atual n\u00E3o \u00E9 mais v\u00E1lida.
+proceedWithAction=&raquo; Clique aqui para continuar
+
+requiredAction.CONFIGURE_TOTP=Configurar OTP
+requiredAction.terms_and_conditions=Termos e Condi\u00E7\u00F5es
+requiredAction.UPDATE_PASSWORD=Atualizar Senha
+requiredAction.UPDATE_PROFILE=Atualizar Perfil
+requiredAction.VERIFY_EMAIL=Verificar E-mail
+
+doX509Login=Voc\u00EA ser\u00E1 logado como\:
+clientCertificate=Certificado de cliente X509\:
+noCertificate=[Sem Certificado]
+
+
+pageNotFound=P\u00E1gina n\u00E3o encontrada
+internalServerError=Ocorreu um erro interno no servidor
+
+console-username=Nome do usu\u00E1rio:
+console-password=Senha:
+console-otp=C\u00F3digo \u00FAnico:
+console-new-password=Nova Senha:
+console-confirm-password=Confirma\u00E7\u00E3o de senha:
+console-update-password=\u00C9 necess\u00E1rio atualizar sua senha.
+console-verify-email=Voc\u00EA precisa verificar seu endere\u00E7o de e-mail.  Enviamos um email para {0} que cont\u00E9m um c\u00F3digo de verifica\u00E7\u00E3o.  Digite este c\u00F3digo no campo abaixo.
+console-email-code=C\u00F3digo do email:
+console-accept-terms=Aceita os termos? [s/n]:
+console-accept=s
+
+# Openshift messages
+openshift.scope.user_info=Informa\u00E7\u00E3o do usu\u00E1rio
+openshift.scope.user_check-access=Informa\u00E7\u00E3o do acesso do usu\u00E1rio
```

Comment:
I think `Informações de acesso do usuário` is better.

Reply:
Yes, it is. thanks